c++/src/kj/compat/BUILD

```diff
@@ -0,0 +1,118 @@
+exports_files(["gtest.h"])
+
+cc_library(
+    name = "kj-tls",
```

Comment:
In general target names are preferred to be snake case.

https://bazel.build/rules/build-style#target-naming

Reply:
I wouldn't consider it a blocker, though it would be nice to stick to conventions where possible. Otherwise the project will end up with snake case in some places and kebab case in others. Consistency is always nice.